tests/test_catalog.py

```diff
@@ -205,50 +151,6 @@ def test_search_usagetype(catalog_usagetype_mock):
     assert all(search_results["up42:usageType"].apply(lambda x: params["result1"] in x or params["result2"] in x))
 
 
-@pytest.mark.skip(reason="Flaky catalog return")
-@pytest.mark.live
-@pytest.mark.parametrize(
-    "usage_type,result,result2",
-    [
-        (["DATA"], "DATA", ""),
-        (["ANALYTICS"], "ANALYTICS", ""),
-        (["DATA", "ANALYTICS"], "DATA", "ANALYTICS"),
-    ],
-)
-def test_search_usagetype_live(catalog_live, usage_type, result, result2):
-    """
-    Result & Result2 are one of the combinations of
-    "DATA" and "ANALYTICS". Result2 can be None.
-
-    The result assertion needs to allow multiple combinations,
-    e.g. when searching for ["DATA", "ANALYTICS"],
-    the result can be ["DATA"],
-    ["ANALYTICS"] or ["DATA", "ANALYTICS"].
-    """
-    search_parameters = catalog_live.construct_search_parameters(
-        start_date="2014-01-01T00:00:00",
-        end_date="2020-12-31T23:59:59",
-        collections=["phr"],
-        limit=100,
-        usage_type=usage_type,
```

Comment:
i wasn't aware of this `usage_type` param, didn't find it in the code reference docs. But anyways, will add it to my next PR